resource-metadata/hypothesis.md

```diff
@@ -0,0 +1,139 @@
+# Resource Metadata Model
+
+## Background
```

Comment:
High level suggestion: What people often conflate is metadata that's loosely coupled with a resource for the purpose of extending interaction possibilities or certain behaviours that comes out of it from the kind of data that extends the description of a resource. Making that distinction up front may help. The classic example has to do with how to deal with providing a human-readable label for a container (or from LDP, updating "minimal-container triples"). That use case or anything that's about self-describing resources is outside the scope of "resource metadata" (as I see it and hopefully I'm not off the mark with what the panel intends to do). What's described here is really about the ones that's highlighted: access control, annotation, shapes, server-managed, configuration, and possibly others defined elsewhere. This is important because we still want the primary resource to be useful in its own right without forcing the client to navigate to the the related meta resource (eg. just to get a hold of container's label) - that's besides annotating non-RDF Sources. That also opens up the possibility to progressive enhance so to speak, and if so desired.

It may help to just change the order of information in the Background.

I sense some redundancy with the two example blocks. Perhaps that can be a single list that's tied to use cases eg. at least one for each metadata type.

The section should be condensed as much as possible. It could just be an "Overview" - let's see..

Reply:
Incorporated suggestions and condensed overview.